### PR TITLE
fix: set profile accent color when navigating to profile

### LINKED ISSUE #53: [Profile Export] Color Accent not respected
### Describe the Bug

When importing a profile that uses the Green accent color, the color defaults back to blue. It appears the setting is not respected during export. 

### Expected Behavior

Respect the accent color

### To Reproduce

- Create profile
- Change color to green
- Export profile
- Import profile

### What operating system are you using?

macOS

### What browser are you using?

Chrome

### What version of Node.js are you using?

_No response_

### PR DESCRIPTION
**Pull request by dated**
## Summary

Fixes #53.

## Checklist

-   [x] My changes look good in both light AND dark mode
-   [x] The change is not hardcoded to a single network, but has multi-asset in mind
-   [x] I checked my changes for obvious issues, debug statements and commented code
-   [ ] Documentation _(if necessary)_
-   [ ] Tests _(if necessary)_
-   [x] Ready to be merged

### CODE REVIEW

File: src/domains/profile/hooks/use-profile-import.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ export const useProfileImport = ({ env }: { env: Environment }) => {
 		let profile: Contracts.IProfile;
 
 		try {
+			console.log(profileData);
```
Comment (goga-m) [inline src/domains/profile/hooks/use-profile-import.ts:15]:
```suggestion
```

**Comment by goga-m** [comment]
@dated  do you think it woudl be necessary to also add  the accent color when rendering the import form step? :thinking: 

Something like:
```jsx
	useEffect(() => {
		setProfileAccentColor(profile);
	}, []);
```
in https://github.com/ArdentHQ/arkvault/blob/fix/set-accent-color/src/domains/profile/pages/ImportProfile/ProfileFormStep.tsx#L55

File: src/domains/profile/hooks/use-profile-import.ts (hunk 1 above)
Comment (dated) [reply src/domains/profile/hooks/use-profile-import.ts]: :hide:

**Comment by dated** [comment]
Sounds good @goga-m, I created a dedicated issue for it 👍🏻

**Review by goga-m — APPROVED** [review]

**Review by samharperpittam — APPROVED** [review]

**Pull request merged**